apps/condo/domains/contact/access/Contact.js

```diff
@@ -7,32 +7,49 @@ const get = require('lodash/get')
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById } = require('@open-condo/keystone/schema')
 
+const {
+    serviceUserCanReadSchemaObjectsIfOrganizationConnectedToLinkedB2BApp,
+    serviceUserCanManageSchemaObjectsIfOrganizationConnectedToLinkedB2BApp,
```

Comment:
I think util names are too short, may be we should create longer names?)

Reply:
I tried to make them shorter, but then it becomes more difficult to understand what is happening in them))